Main.dart

```diff
@@ -0,0 +1,36 @@
+void main()
+{
+   Pessoa p1 = Pessoa();
+  p1.nome = "Pedro";
+  p1._altura = 1.76;
+  p1._niver =  DateTime.parse("2003-06-12");
+ print(p1.dados());
+
+}
+
+class Pessoa
+{
+
+   String _nome;
+  double _altura;
+  DateTime _niver;
+  
+  
+  set nome(String nome){this._nome = nome;}
+  String get nome{return this._nome;}
+  
+   int idade(){return DateTime.now().year- niver.year;}
+  
+  set altura(double altura){this._altura = altura;}
```

Comment:
se for criar o encapsulamento apenas pra falar q nem, não há necessidade.
Tem que ser algum caso real, só que nesse exemplo, dá pra vc receber esse valor via construtor e nem precisar de encapsular aqui